backend/game/dice/dice_resolver.py

```diff
@@ -31,12 +33,18 @@ def calculate_heal_from_dice(dice_counter):
     return dice_counter[DieValue.HEAL]
 
 
-def dice_resolution(dice, player):
+def dice_resolution(dice, player, other_players):
     dice_counter = get_dice_count(dice)
     victory_points = calculate_victory_points_from_dice(dice_counter)
     health = calculate_heal_from_dice(dice_counter)
     heal_self_from_dice(player, health)
+
     attack = calculate_attack_from_dice(dice_counter)
+    attackable_players = get_attackable_players(player, other_players)
+    attack_players(player, attackable_players, attack)
+    if attack > 1:
+        move_to_tokyo_if_empty(player, other_players)
+
     energy = calculate_energy_from_dice(dice_counter)
     return victory_points, health, attack, energy
```

Comment:
I had originally envisioned this method to just returning the calculate values and then a different method would do like attack_resolution as it seems to be doing a lot in just this one method